Makefile

```diff
@@ -34,7 +34,7 @@ test-data:
 	rm -rf $(testDataDir)
 	mkdir -p $(tempDir)
 	git clone -b ${branchName} --depth 1 --single-branch ${githubRepoLink} ${gitDataDir}
-	cp ${gitDataDir}rac-experiments-v3.json ${testDataDir}
+	cp ${gitDataDir}rac-experiments-v3-obfuscated.json ${testDataDir}
```

Comment:
Replaced to use the obfuscated mock RAC file

Reply:
I wonder whether the RAC should use a protobuf encoding to reduce size

Reply:
Good idea to consider this - I had forgotten this was possible as last time I used protobufs the encoding was still in json (by accident!); using the native wire format would be a nice way to solve for this.